Library/Formula/med.rb

```diff
@@ -0,0 +1,20 @@
+class Med < Formula
+  desc "Model for Exchange of Data. Data exchange system for numerical models"
+  homepage "http://www.salome-platform.org/user-section/about/med"
+  url "http://files.salome-platform.org/Salome/other/med-3.0.8.tar.gz"
+  sha256 "0f97e9b5ae514aa53f4c0507f1002a3f520c3b959e756c38cfd9cae4a31a0689"
+
+  depends_on "cmake" => :build
+  depends_on "hdf5" => "with-mpi"
```

Comment:
Med should be in Science but I don’t know how to put it there, advice would be great


Professor D M Ingram, PhD
Institute for Energy Systems
School of Engineering
The University of Edinburgh
Colin MacLaurin Road
Edinburgh EH9 3DW

Tel: +44 (0)131 651 9022
School fax: +44 (0)131 650 6554

Latest Papers:
L. Cradden, C. Kalgoeri, I. Martinez Barrios, G. Galanis, D. M. Ingram, and G. Kallos. Multi-criteria site selection for offshore renewable energy platforms. Renewable Energy, 87:791–806, 2016.
http://dx.doi.org/10.1016/j.renene.2015.10.035

Watanabe Y, Ingram DM (2015) Transverse instabilities of ascending planar jets formed by wave impacts on vertical walls. Proc. R. Soc. A 471:20150397.  http://dx.doi.org/10.1098/rspa.2015.0397

The University of Edinburgh is a charitable body, registered in Scotland, with registration number SC005336.

> On 26 Feb 2016, at 18:51, Dominyk Tiller <notifications@github.com> wrote:
> 
> In Library/Formula/med.rb <https://github.com/Homebrew/homebrew/pull/49566#discussion_r54284626>:
> 
> > @@ -0,0 +1,20 @@
> > +class Med < Formula
> > +  desc "Model for Exchange of Data. Data exchange system for numerical models"
> > +  homepage "http://www.salome-platform.org/user-section/about/med"
> > +  url "http://files.salome-platform.org/Salome/other/med-3.0.8.tar.gz"
> > +  sha256 "0f97e9b5ae514aa53f4c0507f1002a3f520c3b959e756c38cfd9cae4a31a0689"
> > +
> > +  depends_on "cmake" => :build
> > +  depends_on "hdf5" => "with-mpi"
> You can't have mandatory cross-tap dependencies in the core (hdf5 is from Science). If it cannot be optional you will need to file this formula over at the Science tap.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Homebrew/homebrew/pull/49566/files#r54284626>.
> 


The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.